src/cata_tiles.cpp

```diff
@@ -1125,7 +1125,7 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
     }
 
     if (it == tile_ids.end()) {
-        long sym = -1;
+        wchar_t sym = -1;
```

Comment:
the warning in question is for comparisons. I don't think there's an equivalent warning for assignments. implicit casting should function appropriately here. further down sym is only used if it's >0 and <256 so the only way -1 would be a problem here is if it wchar_t is an unsigned byte, and we've got much bigger problems if that happens.